bubble_sort.rb

```diff
@@ -0,0 +1,39 @@
+def bubble_sort(array)
+  repeats = (array.length - 1)
+  loop do
+    sorted = false
+    repeats.times do |pos|
+      if array[pos] > array[pos + 1]
+        array[pos], array[pos + 1] = array[pos + 1], array[pos]
+        sorted = true
+      end
+    end
+    break if sorted == false
+  end
+  p array
```

Comment:
- [ ] Notice that normal methods only return the needed value, and if the user wants to print, they do it on their own. So your method only has to return the value. Please, delete that `p` and just return `array` (same in line 28)